src/components/IconButton/IconButton.visual.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { IconButton } from './';
+import { ReactComponent as ShareIcon } from '../../assets/icons/Share.svg';
+
+class IconButtonVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    return (
+      <IconButton
+        icon={<ShareIcon />}
+        data-hook={'storybook-e2e-IconButton'}
+        {...this.props}
+      />
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
```

Comment:
Let's add a test for the disabled state.
You can see how we did it in the `LikeButton` or `IconToggle`